node/src/RedisClient.ts

```diff
@@ -102,7 +102,14 @@ export class RedisClient extends BaseClient {
      *      If the transaction failed due to a WATCH command, `exec` will return `null`.
      */
     public exec(transaction: Transaction): Promise<ReturnType[] | null> {
-        return this.createWritePromise(transaction.commands);
+        return this.createWritePromise<ReturnType[] | null>(
```

Comment:
Should be there `set` as a return type too?

Reply:
there is, I didnt see that before